app/models/pame_evaluation.rb

```diff
@@ -89,14 +89,14 @@ def self.parse_filters(filters)
       when 'iso3'
         countries = options
         site_ids << countries.map{ |iso3| Country.find_by(iso_3: iso3).protected_areas.pluck(:id) }
-        where_params[:sites] = options.empty? ? nil : "protected_area_id IN (#{site_ids.join(',')})"
+        where_params[:sites] = site_ids.empty? ? nil : "protected_area_id IN (#{site_ids.join(',')})"
```

Comment:
`sites_ids` and `country_ids` can be an array of arrays, so `.empty?` will return false in that case.
You would probably need to flatten those variables first.

Reply:
Thanks I will flatten those.